modin/pandas/test/test_io.py

```diff
@@ -456,6 +457,104 @@ def test_from_json_lines():
     teardown_json_file()
 
 
+def test_from_json_string():
+    json_string = """[{"project": "modin"}]"""
+    with pytest.warns(UserWarning):
+        modin_df = pd.read_json(json_string)
+    df_equals(modin_df, pandas.read_json(json_string))
+
+    json_string = """{
```

Comment:
Maybe it’s possible not to use such a complicated string for the test ? After all, we are not using a custom parser :)

Reply:
When we do implement a custom parser I would like to still have this test in that case.

Reply:
I did move the string data into the utils of the tests files where other tests datasets live and parametrized based on your other comment. Let me know what you think!

Reply:
It's good decision. Another suggestion if you don't mind :)